Update violations page info (#91288)
openshift/openshift-docs: modules/analyzing-all-violations.adoc

.Procedure

. In the {product-title-short} portal, click *Violations*.
. Click the tab to view violations for the following categories:
** *User Workloads*: Displays violations for user-managed workloads.
** *Platform*: Displays violations for workloads used by {ocp} and layered services.
** *All violations*: Displays violations for user workload and platform components, and audit log violations for cluster resources.

> . To view different types of violations, click one of the following tabs:

Reviewer: @sr1kar99 The original sentence adheres to [ISG](https://www.ibm.com/docs/en/ibm-style?topic=format-procedures#writing-steps).

** *Active*: Displays violations in the build and deploy stages or unresolved runtime violations.
** *Resolved*: Displays the following violations:
*** Violations in the build and deploy stages for workloads that were removed or modified to be compliant
*** Manually resolved runtime violations
*** Violations in all stages that were generated before a policy exclusion was added

> ** *Attempted*: Displays violations for deployment actions that were attempted but blocked by a pre-check of {product-title-short} policies by the {product-title-short} admission controller. For example, the deployment action that was attempted would have triggered a violation if it had been allowed to succeed, but the policy enforcement behavior did not allow it to succeed.

Reviewer 2: blocked by evaluation of enforced policies

... but the admission controller enforcement did not allow the operation to be admitted to the cluster.

> ** *Attempted*: Displays violations for deployment actions that were attempted but blocked by the evaluation of enforced policies. For example, the admission controller found that the attempted deployment action, such as a deployment create, update, or scale, would have triggered a violation. The admission controller enforcement then did not allow the operation to be admitted to the cluster.

Reviewer 3: ```suggestion
** *Attempted*: Displays violations for deployment actions that were attempted but blocked by enforced policies. For example, if the admission controller detected that creating, updating, or scaling a deployment would trigger a violation, it prevents the operation from running in the cluster.
```

Feel free to disagree. :)

Author: Good suggestion - am keeping the examples because I think there could be other actions that might trigger a violation other than these?

> . Optional: Choose the appropriate method to reorganize or filter information in the *Violations* page:

Reviewer 3: ```suggestion
. Optional: Choose a method to reorganize or filter information on the *Violations* page:
```
